### ISSUE TITLE
Please add a written getting started guide

### ISSUE THREAD

**Issue by ukch** [atlassian/react-beautiful-dnd#1779]
Dear all,
I love the look of this library. However, your written documentation is sorely lacking code samples. I see you have a Netlify demo app, and some getting started videos, but there is no obvious way to view the code for the demos, and videos are just not a helpful way to learn for many people. Please can you add a written 'getting started' guide, or some basic tutorials?

As an example from the competition, please check out the excellent tutorial from [react-dnd](https://react-dnd.github.io/react-dnd/docs/tutorial).

**Comment by Mulli** [comment]
Try https://egghead.io/lessons/react-reorder-a-list-with-react-beautiful-dnd
Please note that if first dragging example is not working (I am using latest react) - 
replace ```innerRef = {..}``` with ```ref={...}```
Enjoy

**Comment by ukch** [comment]
Thanks @Mulli, but that is not a **written** guide, so not really relevant to the point I'm trying to make here.

**Comment by mcadam** [comment]
Yep the lessons are not the best for this kind of component, I just want a quick started guide without having to go through dozens of videos.

**Comment by oyeanuj** [comment]
In addition, Egghead does not let you access the code in the samples unless you are a premium member.

**Comment by samirelanduk** [comment]
100% agree - I can't really get to grips with this library without a written Getting Started. A video tutorial is a great complement to written documentation, but having it be *instead* of written documentation is very frustrating.
